roles/aodh/tasks/main.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Check if aodh user exists
+  action: shell getent passwd aodh
+  register: aodh_user
+  failed_when: False
+  changed_when: False
+
+- name: aodh user
+  user: name=aodh shell=/bin/false createhome=no
+  when: aodh_user|success
```

Comment:
The user module is idempotent, why are we checking first and adding after? Why not just use the user module?